centralized-sampling-tests/integration-tests/src/main/java/com.amazon.tests/SampleRules.java

```diff
@@ -0,0 +1,253 @@
+package com.amazon.tests;
+
+import org.json.simple.JSONObject;
+
+/** File that contains all sample Rules that will be created to be used for testing */
+public class SampleRules {
+  public String name;
+  public String JSON;
+
+  public double ExpectedSampled;
+
+  /**
+   * @param name - name of the sample rule
+   * @param JSON - JSON object to send to the xray to create sample rule
+   * @param ExpectedSampled - rate of expected sampling
+   */
+  public SampleRules(String name, String JSON, double ExpectedSampled) {
+    this.JSON = JSON;
+    this.name = name;
+    this.ExpectedSampled = ExpectedSampled;
+  }
+
+  /**
+   * Function to get a string JSON variable to send to xray to create a sample rule
+   *
+   * @param name - String - name of the rule
+   * @param priority - int - priority of the rule
+   * @param reservoir - int - reservoir or minimum it should sample
+   * @param rate - double - fixed rate of the sample rule
+   * @param serviceName - String - service name to filter by, "*" if none
+   * @param method - String - REST method to filter by, "*" if none
+   * @param path - PATH - path to filter by, "*" if none
+   * @param attributes - JSONObject - attributes to filter by, can be null
+   * @return
+   */
+  public static String getJSON(
+      String name,
+      int priority,
+      int reservoir,
+      double rate,
+      String serviceName,
+      String method,
+      String path,
+      JSONObject attributes) {
+    JSONObject jsonObject = new JSONObject();
+    JSONObject jsonBody = new JSONObject();
+    jsonBody.put("FixedRate", rate);
+    jsonBody.put("Host", "*");
+    jsonBody.put("HTTPMethod", method);
+    jsonBody.put("Priority", priority);
+    jsonBody.put("ReservoirSize", reservoir);
+    jsonBody.put("ResourceARN", "*");
+    jsonBody.put("RuleName", name);
+    jsonBody.put("ServiceName", serviceName);
+    jsonBody.put("ServiceType", "*");
+    jsonBody.put("URLPath", path);
+    jsonBody.put("Version", 1);
+    if (attributes != null) {
+      jsonBody.put("Attributes", attributes);
+    }
+    jsonObject.put("SamplingRule", jsonBody);
+
+    return jsonObject.toString();
+  }
+
+  /**
+   * Sample rule that samples all targets
+   *
+   * @return AcceptAll SampleRule
+   */
+  private static SampleRules getAcceptAll() {
+    String JSON = SampleRules.getJSON("AcceptAll", 1000, 1, 1.0, "*", "*", "*", null);
```

Comment:
It would probably be better to use the Builder pattern instead of this collection of parameters. That way when we're getting a particular JSON object, we can see which data types each input map to clearly. For example:

```java
String sampleRuleJson = SampleRuleBuilder
  .withName("AcceptAll")  // These return the buil
  .withPriority(1000)
// ...
  .toJson();  // This actually converts it to a JSON string
```

See [this guide](https://howtodoinjava.com/design-patterns/creational/builder-pattern-in-java/)